sem/dashboard/__init__.py

```diff
@@ -0,0 +1,59 @@
+from flask import Flask, render_template, jsonify, request
+
+from sem.dashboard.dashboard import Dashboard
+
+
+flask_app = Flask(__name__)
+dashboard = Dashboard()
+
+
+@flask_app.route("/")
+def index():
+    return render_template("dashboard.html")
+
+
+@flask_app.route("/serverside_table", methods=['GET'])
+def serverside_table_content():
+    dashboard.set_request(request)
+    data = dashboard.build_datatable()
+    return jsonify(data)
+
+
+@flask_app.route("/filters", methods=['GET'])
+def filter():
+    dashboard.set_filter_request(request)
+
+    data = dashboard.buildchart()
+    plot_data = []
+    for i in data:
+        plot_data += [{'x': i['time'], 'y': float(i['jitter_context'])}]
```

Comment:
check [this](https://stackoverflow.com/questions/30245397/why-is-a-list-comprehension-so-much-faster-than-appending-to-a-list) out
```suggestion
    plot_data = [{'x': row['time'],
                         'y': float(row['jitter_context'])}
                         for row in data]
```